<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Фоторедактор</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #3b3938; color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>721</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="main_layout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="topMargin">
      <number>30</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <property name="bottomMargin">
      <number>30</number>
     </property>
     <item>
      <widget class="QLabel" name="label_image">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #3b3938; color: white;</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="action_rectangle"/>
   <addaction name="action_circus"/>
   <addaction name="action_line"/>
   <addaction name="action_text_2"/>
   <addaction name="action_color_fill"/>
   <addaction name="action_outline"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>797</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #5c5a59; color: white;</string>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="action_save_as"/>
    <addaction name="action_close"/>
    <addaction name="action_properties"/>
    <addaction name="action_settings"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Редактирование</string>
    </property>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>Поворот</string>
     </property>
     <addaction name="separator"/>
     <addaction name="action_rotate_clockwise"/>
     <addaction name="action_rotate_counterclockwise"/>
     <addaction name="action_rotate_arbitary"/>
    </widget>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>Отражение</string>
     </property>
     <addaction name="action_reflect_horizontal"/>
     <addaction name="action_reflect_vertical"/>
    </widget>
    <addaction name="action_size"/>
    <addaction name="action_crop"/>
    <addaction name="menu_5"/>
    <addaction name="menu_6"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Фильтры</string>
    </property>
    <addaction name="action_without_filters"/>
    <addaction name="action_grey"/>
    <addaction name="action_filter_sepia"/>
    <addaction name="action_3d"/>
    <addaction name="action_texture"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_circus"/>
    <addaction name="action_line"/>
    <addaction name="action_text_2"/>
    <addaction name="action_color_fill"/>
    <addaction name="action_outline"/>
   </widget>
   <widget class="QMenu" name="menu_7">
    <property name="title">
     <string>Эффекты</string>
    </property>
    <addaction name="action_brightness"/>
    <addaction name="action_opacity"/>
    <addaction name="action_warm"/>
    <addaction name="action_blur"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_7"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <action name="action_open">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="action_crop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Обрезать</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_grey">
   <property name="text">
    <string>Градации серого</string>
   </property>
  </action>
  <action name="action_properties">
   <property name="text">
    <string>Свойства</string>
   </property>
  </action>
  <action name="action_size">
   <property name="text">
    <string>Размер</string>
   </property>
  </action>
  <action name="action_close">
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="action_filter_sepia">
   <property name="text">
    <string>Сепия</string>
   </property>
  </action>
  <action name="action_filter_warm">
   <property name="text">
    <string>Тёплый</string>
   </property>
  </action>
  <action name="action_filter_cold">
   <property name="text">
    <string>Холодный</string>
   </property>
  </action>
  <action name="action_filter_tape">
   <property name="text">
    <string>Плёнка</string>
   </property>
  </action>
  <action name="action_circus">
   <property name="text">
    <string>Круг</string>
   </property>
  </action>
  <action name="action_line">
   <property name="text">
    <string>Линия</string>
   </property>
  </action>
  <action name="action_text_2">
   <property name="text">
    <string>Текст</string>
   </property>
  </action>
  <action name="action_color_fill">
   <property name="text">
    <string>Цвет заливки</string>
   </property>
  </action>
  <action name="action_rotate_clockwise">
   <property name="text">
    <string>90° по ч. с.</string>
   </property>
  </action>
  <action name="action_rotate_counterclockwise">
   <property name="text">
    <string>90° против ч. с.</string>
   </property>
  </action>
  <action name="action_rotate_arbitary">
   <property name="text">
    <string>Произвольно</string>
   </property>
  </action>
  <action name="action_reflect_horizontal">
   <property name="text">
    <string>По горизонтали</string>
   </property>
  </action>
  <action name="action_reflect_vertical">
   <property name="text">
    <string>По вертикали</string>
   </property>
  </action>
  <action name="action_outline">
   <property name="text">
    <string>Обводка</string>
   </property>
  </action>
  <action name="action_without_filters">
   <property name="text">
    <string>Без фильтров</string>
   </property>
  </action>
  <action name="action_brightness">
   <property name="text">
    <string>Яркость</string>
   </property>
  </action>
  <action name="action_opacity">
   <property name="text">
    <string>Прозрачность</string>
   </property>
  </action>
  <action name="action_warm">
   <property name="text">
    <string>Теплота</string>
   </property>
  </action>
  <action name="action_3d">
   <property name="text">
    <string>3d</string>
   </property>
  </action>
  <action name="action_texture">
   <property name="text">
    <string>Текстура</string>
   </property>
  </action>
  <action name="action_rectangle">
   <property name="text">
    <string>Прямоугольник</string>
   </property>
  </action>
  <action name="action_blur">
   <property name="text">
    <string>Размытие</string>
   </property>
  </action>
  <action name="action_settings">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
